pages/index.js

```diff
@@ -36,6 +36,7 @@ import {
   UserCircleIcon,
   XMarkIcon,
 } from "@heroicons/react/24/outline";
+import { data } from "autoprefixer";
```

Comment:
Wgat is this for? I don't see it used anywhere?

Reply:
I have no idea. I've never seen this before. I'll remove it now.